plugins/plugin-svelte/plugin.js

```diff
@@ -45,17 +45,10 @@ module.exports = function plugin(snowpackConfig, pluginOptions = {}) {
           })
         ).code;
       }
-      // COMPILE
-      const ssrOptions = {};
-      if (isSSR) {
-        ssrOptions.generate = 'ssr';
-        ssrOptions.hydratable = true;
```

Comment:
Ah, I must have misunderstood this. Isn't the idea that `hydratable` is needed in SSR mode so that it generated hydratable markup, so that the client-side code can hydrate the SSR-generated markup?

Reply:
No, `hydratable` just means that the client-side component is generated with the code necessary to hydrate the server-rendered markup. There aren't any constraints on the markup that it hydrates, it will repair the DOM as necessary